### DISCUSSION TITLE
please open this page

### DISCUSSION THREAD

**Discussion by behrouz1** [community/community#38501] [Other Feature Feedback, Questions, & Ideas]
Hi. please open this page
https://github.com/gildas-lormeau/SingleFile/issues/1039

I don't another way to contact . tanks

**Comment by david-engelmann** [comment]
It would be best to resubmit the issue through [the SingleFile repo issue section](https://github.com/gildas-lormeau/SingleFile/issues). This discussion section is mainly for topics related to Github and not individual repositories. Best of Luck!
